tests/find_all_pytests.py

```diff
@@ -0,0 +1,22 @@
+"""Find all pytests in the tests directory."""
```

Comment:
What's the purpose of this script? It doesn't see to be used and it seems a simple
```bash
find ./tests -name 'test_*.py'
```
does the same.

Reply:
cheers Klaus! `find` has never actually fully worked for me - it always missed one or two or more files - so I gave up using it a while ago. But I can defo remove that script (especially since it's untested) if you reckon so :+1: